internal/relationtuple/manager_isolation.go

```diff
@@ -0,0 +1,75 @@
+package relationtuple
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func twice(t *testing.T, m0, m1 Manager) func(string, func(*testing.T, Manager, Manager)) {
+	return func(desc string, run func(*testing.T, Manager, Manager)) {
+		t.Run(desc+"-0", func(t *testing.T) {
+			run(t, m0, m1)
+		})
+		t.Run(desc+"-1", func(t *testing.T) {
+			run(t, m1, m0)
+		})
+	}
+}
+
+func IsolationTest(t *testing.T, m0, m1 Manager, addNamespace func(context.Context, *testing.T, string)) {
```

Comment:
Regarding isolation, what cases are missing in this test in your opinion @aeneasr? Or how should the isolation test work instead of this?

Reply:
This tests namespace isolation, but not network isolation, right?

Reply:
^ @zepatrik

Reply:
https://github.com/ory/keto/pull/638#discussion_r669556863